src/pages/TeamPage/index.js

```diff
@@ -0,0 +1,348 @@
+import React, { useState, useEffect } from "react";
+import {
+  Button,
+  Typography,
+  Box,
+  Container,
+  Stack,
+  CssBaseline,
+  Card,
+} from "@mui/material";
+import { Grid } from "@mui/material";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import useStyles from "./style";
+import axios from "axios";
+import Tippy from "@tippyjs/react";
+import "./style.css";
+import { breakpoints } from "../../theme/constant";
+import LinkedIn from "../../components/common/SocialMediaIcons/LinkedIn";
+import Twitter from "../../components/common/SocialMediaIcons/Twitter";
+
+function shuffleObject(obj) {
+  let newObj = {};
+  var keys = Object.keys(obj);
+  keys.sort(function (a, b) {
+    return Math.random() - 0.5;
+  });
+  keys.forEach(function (k) {
+    newObj[k] = obj[k];
+  });
+  return newObj;
+}
```

Comment:
There are a few issues with this function.  The first is that the properties of objects with non-numerical keys don't have an inherent order.  Instead, you're attempting to shuffle the **insertion order** of the key-value pairs and are relying on `Object.keys` to retain this order.  But if the object were an Array (keys are the indices), nothing would get changed as `Object.keys` would list the indices in increasing order.  Similarly, if you have some keys that are nonnegative integers and others not, the nonegative integer keys will always appear first.  For example:
```
const obj = {};
obj['foo'] = 'something';
obj[42] = 'bar';
console.log(Object.keys(obj)); // logs 42. then foo
```
Even if you only have keys that are strings, the situation with `Object.keys` preserving insertion order is more complicated prior to ES2020 (See: https://stackoverflow.com/a/30919039).

Another issue is that the shuffling algorithm will be biased and have time complexity O(nlgn) (assuming the sort has this complexity but this is an implementation detail: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) instead of O(n), which will make it a little more inefficient for larger Arrays.

Finally, you're only using the Array `Object.keys` so it would make more sense to return that rather than the object itself.

So to address these issues, I'd use the standard [Fisher-Yates shuffle algorithm](https://en.wikipedia.org/wiki/Fisher%E2%80%93Yates_shuffle) which takes an Array as an input, shuffles it, and returns the shuffled Array as an output all in linear time.  Additionally, I would put this function in `utils.js` as it is a general function that's not specific to this component and could be used elsewhere so this could be the function to export:
```
const shuffleArr = (arr) => {
  for(let i = arr.length-1; i >= 0; i--) {
    const newIndex = Math.floor(Math.random()*(i+1));
    [arr[i], arr[newIndex]] = [arr[newIndex], arr[i]]
  }
  return arr;
};
```
This essentially follows the standard [Fisher-Yates shuffle algorithm](https://en.wikipedia.org/wiki/Fisher%E2%80%93Yates_shuffle) on an Array, but it's creating a new Array instead of shuffling in place.